### PR TITLE
Post using async and await with ember-cli-page-object

### PR DESCRIPTION
**Pull request by san650**

### CODE REVIEW

File: source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+  await page
+    .password('secret')
+    .submit();
+
+  assert.equal(page.message, 'Welcome back John Doe');
+});
+{% endcodeblock %}
+
+As you can see [ember-cli-page-object](https://github.com/san650/ember-cli-page-object)'s actions support the `await` keyword, even if you chain multiple actions. Also note that you have to mark the test function as asynchronous `async function(assert)`.
```
Comment (jmbejar) [inline source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown]: Maybe adding a comma after "As you can see" ?

**Comment by jmbejar** [comment]
Nice post!!! 👏

**Comment by spastorino** [comment]
Very nice, concise and well written post 👏.
What about adding a final paragraph with a very little conclusion or final thought?.

File: source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown [hunk 2]
```diff
@@ -111,10 +111,16 @@ Take a look at [this commit](https://github.com/san650/tajpado/commit/6638a26564
 
 ## A note about JSHint
 
-JSHint doesn't like the new _async_ and _await_ keywords so it returns a handfull of errors. To disable JSHint on a particular file you can add a comment directive on top of the file.
+JSHint doesn't like the new keywords _async_ and _await_ so it returns a handfull of errors. To disable JSHint on a particular file you can add a comment directive on top of the file.
```
Comment (spastorino) [inline source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown]: handfull => handful

Comment (spastorino) [inline source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown]: And in the whole change I think keywords goes at the end. At least it should be consistent with https://github.com/wyeworks/blog/pull/35/commits/d637cb9eb4fdbc73d2eecef609bd7266f003ef21#diff-881e145733de402e9a68e44394539addR20 :)

File: source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown [hunk 3]
```diff
@@ -111,10 +111,16 @@ Take a look at [this commit](https://github.com/san650/tajpado/commit/6638a26564
...
 /* jshint ignore:start */
 ```
 
 Note that JSHint will process all files of the project except the ones that contains this comment.
+
+## The future
+
+There's a proposal in Ember.js by [Robert Jackson](https://github.com/rwjblue) (see the [RFC](https://github.com/emberjs/rfcs/pull/119)) to use async functions as the default way to write acceptance test. Although this proposal is in an early stage it seems to have a good reception by the community.
+
+See https://github.com/rwjblue/rfcs/blob/42/text/0000-grand-testing-unification.md#async--await
```
Comment (spastorino) [inline source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown]: :+1:

File: source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown (hunk 1 above)
Comment (san650) [inline source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown]: done

File: source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown (hunk 2 above)
Comment (san650) [inline source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown]: done and done

**Pull request merged**